iam/app.py

```diff
@@ -198,43 +130,4 @@ def set_password(id):
         except NoResultFound:
             print(f"No user has id {id} (try `flask users`)")
 
-    return app
-
-
-def sign_claims(app, user):
-    """return signed jwt and refresh token for the given authenticated user"""
-    user.refresh_token = secrets.token_hex(32)
-    user.refresh_token_expiry = (
-        datetime.now() + app.config['REFRESH_TOKEN_VALIDITY'])
-    db.session.commit()
-    claims = {
-        'exp': int((datetime.now() + app.config['JWT_VALIDITY'])
-                   .strftime('%s'))
-    }
-    claims.update(user.claims)
-    signed_token = jwt.encode(claims, app.config['RSA_PRIVATE_KEY'],
-                              app.config['ALGORITHM'])
-    return {
-        'jwt': signed_token,
-        'refresh_token': {
-            'val': user.refresh_token,
-            'exp': int(user.refresh_token_expiry.strftime('%s')),
-        }
-    }
-
-
-def create_firebase_user(uid, decoded_token):
-    if ' ' in decoded_token['name']:
-        first_name, last_name = decoded_token['name'].split(None, 1)
-    else:
-        first_name, last_name = decoded_token['name'], ''
-
-    user = User(
-        firebase_uid=uid,
-        first_name=first_name,
-        last_name=last_name,
-        email=decoded_token['email'],
-    )
-    db.session.add(user)
-    db.session.commit()
-    return user
+    return application
```

Comment:
Since you only mutate the `app` and `api`, you don't really have to return anything.